src/routes/(console)/project-[region]-[project]/databases/database-[database]/collection-[collection]/document-[document]/attributes/store.ts

```diff
@@ -15,3 +15,8 @@ export function isRelationship(attribute: Attributes): attribute is Models.Attri
     if (!attribute) return false;
     return attribute?.type === 'relationship';
 }
+
+export function isString(attribute: Attributes): attribute is Models.AttributeString {
+    if (!attribute) return false;
+    return attribute?.type === 'string';
+}
```

Comment:
this is only used in the `collection-[collection]/attributes` area. best to move it there if not used in the documents section. Also `store.ts` isn't the best place for this.